docs/_docs/customizingyourgateway.md

```diff
@@ -40,34 +42,35 @@ For example:
 
 ```go
 mux := runtime.NewServeMux(
-	runtime.WithMarshalerOption("application/json+pretty", &runtime.JSONPb{Indent: "  "}),
+  runtime.WithMarshalerOption("application/json+pretty", &runtime.JSONPb{Indent: "  "}),
 )
 prettier := func(h http.Handler) http.Handler {
-	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		// checking Values as map[string][]string also catches ?pretty and ?pretty=
-		// r.URL.Query().Get("pretty") would not.
-		if _, ok := r.URL.Query()["pretty"]; ok {
-			r.Header.Set("Accept", "application/json+pretty")
-		}
-		h.ServeHTTP(w, r)
-	})
+  return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+    // checking Values as map[string][]string also catches ?pretty and ?pretty=
+    // r.URL.Query().Get("pretty") would not.
+    if _, ok := r.URL.Query()["pretty"]; ok {
+      r.Header.Set("Accept", "application/json+pretty")
+    }
+    h.ServeHTTP(w, r)
+  })
```

Comment:
This is still replacing tabs with spaces in this snippet. Is this a formatter in your editor maybe? It might be easiest to start again from scratch, turn off the formatter, and just correct the prose?